source.ipynb

```diff
@@ -49,7 +57,9 @@
     "## Data Sources\n",
     "*What 3 data sources have you identified for this project?*\n",
     "*How are you going to relate these datasets?*\n",
-    "📝 <!-- Answer Below -->"
+    "📝 <!-- Answer Below -->\n",
+    "\n",
+    "The three data sources that I will be using for this project will be the openpowerlifting.csv file (*Had to remove from my data because it exceeded github's size. I have it on my laptop downloads*), the website https://exrx.net/Testing/WeightLifting/DeadliftStandards on all main lifts (Squat, deadlift, and bench), and my own data I have recorded from my lifts as well. I will add my personal data to the powerlifting data to see how my own stats compare to other powerlifters within my weight range. Within the website, I will use the tables to determine where the powerlifters fall into regarding their weight and lifts (If they are itermidiate, advanced, elite, etc.)"
```

Comment:
it's ok you had to remove the data from the project, but the notebook should allow me to download the file from it's original source, so I can replicate that project on my end.